README.md

```diff
@@ -54,6 +55,20 @@ refer to [askubuntu](https://askubuntu.com/questions/766334/cant-login-as-mysql-
 4. Install `docker` and `docker-compose`.
 Please refer to [official doc](https://docs.docker.com/engine/install/).
 
+## Client Testing
+1. Set up the server and client
+
+Launch EVA database Server: `python eva.py`
+
+Launch CLI: `python eva_client.py`
+
+2. Run these 2 commands in the client terminal: (may take a while)
```

Comment:
Change this to try running queries on the client, following are a few queries to start with. Shouldn't look like these are mandatory instructions :p

Reply:
Done